mediapipe/calculators/util/counting_vector_size_calculator.cc

```diff
@@ -0,0 +1,26 @@
+// Copyright 2020 The MediaPipe Authors.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+#include "mediapipe/calculators/util/counting_vector_size_calculator.h"
+
+#include "mediapipe/framework/formats/landmark.pb.h"
+
+namespace mediapipe {
+
+typedef CountingVectorSizeCalculator<
```

Comment:
perhaps just CountLandmarksCalculator?

Reply:
Actually, I take this part from [#867](https://github.com/google/mediapipe/pull/867)

Also, this name is more generic and can be used in other parts of the project, and here I used it for counting detected faces.